_posts/2019-11-06-#15845.md

```diff
@@ -48,9 +48,9 @@ host: jnewbery
 3. How many scripts will a newly created wallet have? How many items will be
    added to the GCS returned by `GetAllRelevantScriptPubKeys()`?
 
-4. What is the false postive rate for Basic compact block filters? How many
-   false positives blocks would you expect a newly created block to retrieve if
-   scanning the entire block-chain? (note that newly-created keys have _birthdays_
+4. What is the false positive rate for Basic compact block filters? How many
+   false positive blocks would you expect a newly created block to retrieve if
+   scanning the entire block-chain? (note that newly created keys have _birthdays_
```

Comment:
(also s/block-chain/block chain/ if following the Optech style guide, but whatev)